runtime/parachains/src/configuration/migration.rs

```diff
@@ -148,15 +148,15 @@ pub mod v4 {
 		}
 
 		fn on_runtime_upgrade() -> Weight {
-			if StorageVersion::get::<Pallet<T>>() == 3 {
```

Comment:
In line 146 above it is checking for version 3.  
No idea why the CI is not catching that, with this [recent change](https://github.com/paritytech/substrate/pull/13684) the pre- and post checks should run. cc @kianenigma

Reply:
oops, looks like I missed changing this and indeed this did not fail ....